### DISCUSSION TITLE
Curious if the benchmarks comparing different libraries is updated automatically?

### DISCUSSION THREAD

**Discussion by kszlim** [coiled/benchmarks#1426] [Q&A]
I was wondering if https://tpch.coiled.io/ reflects the most current state of the benchmarking suite?

Thanks!

**Comment by hendrikmakait** [comment]
Hi, it does not. We have recently added the remaining TPC-H queries and seen quite a few improvements on `dask-expr` in particular. We should update this soon, I'll send an update on this discussion then.

**Reply by kszlim** [reply]
With the query planning turned on by default for dask in the new release, i'm curious if these numbers look very different now? Would be great to see a benchmark!
